src/FixerDescriptionAwareInterface.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer;
+
+/**
+ * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ */
+interface FixerDescriptionAwareInterface
+{
+    /**
+     * @return FixerDescription
+     */
+    public function getFixerDescription();
```

Comment:
>I totally agree that putting a getDesc...DTO... in main interface and enforcing it everywhere would be really nice option,

I disagree, if think having one interface to deal with all type of descriptions of a fixer would be better. Than one can add it to a custom fixer or not, as it is not needed runtime it makes it easier to create a custom fixer.
We can (and should) force that fixers in this project implement the interface.

I think it is better to have one method to get the description of a fixer (i.e. the DTO) and have a method on that object to get the short description, I see no value in having both a DTO object describing a part of the fixer and another method describing another part.

IMHO removing the `getDescription` from the `FixerInterface` is not a big BC break, but sure it is one :)